WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -58,17 +80,25 @@ - (void)loadView
 
     [WPStyleGuide configureColorsForView:self.view andTableView:self.tableView];
 
-    UINib *headerCellNib = [UINib nibWithNibName:@"CommentTableViewHeaderCell" bundle:nil];
-    UINib *bodyCellNib = [UINib nibWithNibName:@"CommentTableViewCell" bundle:nil];
-    [self.tableView registerNib:headerCellNib forCellReuseIdentifier:CVCHeaderCellIdentifier];
-    [self.tableView registerNib:bodyCellNib forCellReuseIdentifier:CVCCommentCellIdentifier];
-    self.headerLayoutCell = [self.tableView dequeueReusableCellWithIdentifier:CVCHeaderCellIdentifier];
-    self.bodyLayoutCell = [self.tableView dequeueReusableCellWithIdentifier:CVCCommentCellIdentifier];
+    // Register Cell Nibs
+    NSArray *cellClassNames = @[
+        NSStringFromClass([NoteBlockHeaderTableViewCell class]),
+        NSStringFromClass([NoteBlockCommentTableViewCell class]),
+        NSStringFromClass([NoteBlockActionsTableViewCell class])
+    ];
+    
+    for (NSString *cellClassName in cellClassNames) {
+        Class cellClass         = NSClassFromString(cellClassName);
+        NSString *className     = [cellClass classNameWithoutNamespaces];
```

Comment:
Catch to it is that we need both, Class with its full namespace (so that we can hit the `reuseIdentifier + layoutIdentifier` helpers), and namespace-less, just to load the nib.

We could get away with that change if, once loaded the nib, we could retrieve (somehow) the cell's namespace. But i've been unable to find any helper to achieve that =(

Another alternative is to do something like this, perhaps:

```
NSDictionary *cellMap = @{
    [NoteBlockHeaderTableViewCell reuseIdentifier] : [NoteBlockHeaderTableViewCell classNameWithoutNamespaces],
    [NoteBlockHeaderTableViewCell layoutIdentifier] : [NoteBlockHeaderTableViewCell classNameWithoutNamespaces],
/// (...)
};
```

And then iterate through each tuple (Key > Value) and register the (Nib) against the reuseIdentifier). What do you think Eric?

Thank you!